clusterloader2/pkg/measurement/common/ooms_tracker.go

```diff
@@ -178,10 +191,19 @@ func (m *clusterOOMsTrackerMeasurement) gather() ([]measurement.Summary, error)
 }
 
 func (m *clusterOOMsTrackerMeasurement) validate() error {
-	if len(m.ooms) == 0 {
-		return nil
+	for _, oom := range m.ooms {
+		var isWhitelisted bool = false
+		for _, whitelistedProcess := range m.oomsWhitelist {
+			if oom.Process == whitelistedProcess {
+				isWhitelisted = true
+				break
+			}
+		}
+		if !isWhitelisted {
+			return fmt.Errorf("OOMs recorded: %+v", m.ooms)
```

Comment:
you should create new, filtered list of ooms and not return full list or nothing

Reply:
Done.